TedTalks/TedTalks/Controllers/TedTalkListViewController.swift

```diff
@@ -52,4 +64,30 @@ extension TedTalkListViewController: UITableViewDelegate, UITableViewDataSource
         showTedTalkDetail(tedTalkManager.tedTalks[indexPath.row])
```

Comment:
This is passing the wrong talk since you are using the unfiltered list here, maybe you want to make that list private so you can't use it by mistake.

Reply:
Thank you, fixed